### DISCUSSION TITLE
Google group banned

### DISCUSSION THREAD

**Discussion by adam12** [jeremyevans/sequel#2145] [General]
The Google group is showing up as banned for me. I find historical access to the group helpful occasionally. Is this something that can be corrected or is it lost?
![image](https://github.com/jeremyevans/sequel/assets/17083/237aeaa5-3f49-4de7-94d4-80dd49351296)

**Comment by jeremyevans** [comment]
I don't think there is anything that can be done.  So I removed all references to it from Sequel's repository a couple weeks ago in e3936630f9b0df561830ca497950ded171137473

**Reply by AlexWayfer** [reply]
@jeremyevans probably, you can try this: https://support.google.com/groups/answer/14187429?hl=en&sjid=12568721133455252117-EU

**Reply by jeremyevans** [reply]
Thanks, I tried that.  Let's see if it makes a difference.

**Reply by jeremyevans** [reply]
Apparently not.  Here's the response:

```
Hello [email],

We're letting you know that we've permanently removed the content at https://groups.google.com/d/forum/sequel-talk

Why did this happen?

An external report flagged the content for illegal content or policy violations. As a result, our legal content and policy standards team removed the content for the following reason: regulated.

Learn more

You can learn more about our content policies and enforcement at our [help center](https://support.google.com/groups/answer/4561696?hl=en_US).

What you can do next

You may have the option to pursue your claims in court. If you have legal questions or wish to examine legal options that may be available to you, you may want to consult with your own legal counsel.
```

**Reply by adam12** [reply]
Wow. That's horrible. There was a treasure trove of prior suggestions and discussions that were incredibly valuable. I referenced it a few times a week sometimes.

I'm assuming you still don't have access as an Admin?

**Reply by jeremyevans** [reply]
I can still see all of the content.  So "permanently removed" doesn't seem to be accurate.

**Reply by adam12** [reply]
I thought maybe we could run google-group-crawler on it and just have an mbox archive somewhere, but that looks to be a dead-end too! I guess there are not many options unless we can get our request to unban escalated somehow. :(

**Reply by jeremyevans** [reply]
I think you are correct.  If anyone has contacts as Google and can escalate, please do.

**Comment by adam12** [comment]
I've noticed a lot of other groups have been blocked recently as well. `git-users@googlegroups` is another popular one.

While I've been poking about, I found that at one point in time, mail-archive was subscribed. So there is a fair amount of email archived. The URL is: https://www.mail-archive.com/sequel-talk@googlegroups.com/

Backscrolling through the date view, it only starts around May 2017, but searching yields content back to 2007. This might serve as a reasonable replacement and perhaps would be valuable to add to the documentation.

Hopefully, mail-archive doesn't disappear.

**Reply by jeremyevans** [reply]
That sounds reasonable.  Can you submit a PR for it?

**Reply by adam12** [reply]
I will submit a PR.
